README.rst

```diff
@@ -65,6 +65,13 @@ Features
 
 -----
 
+=============
+Prerequisites
+=============
+
+* Python >= 3.4
+* pip >= 8.0
```

Comment:
That is weird, because my log shows the author
commit 8b50827bb2b6e44f98535ee89e37cf75c8e2d2c2
Author: Anna Melzer <anna@data-and-dragons.com>
Date:   Sat Oct 29 13:07:34 2016 +0200

    README.rst: Add min pip version
    
    The installation of coala fails if the pip version is too low.
    The requirements for Python and pip have been added to make them
    easy to find.